docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
+    {label:'Seattle', value: 'seattle'},
+  ]}
+  label='City'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList when additional functionality such as subtext or images are needed. Use Dropdown instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'List item 1', value: 'Value'},
+    {label:'I want to put an image here, but I can’t :( ', value: 'Value'},
+    {label:'List item 3', value: 'Value'},
+    {label:'List item 4', value: 'Value'},
+    {label:'List item 5', value: 'Value'},
+  ]}
+  label=''
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Order the list items in SelectList either alphabetically or by usage."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Alberto', value: 'alberto'},
+    {label:'Annie', value: 'annie'},
+    {label:'Ashley', value: 'ashley'},
+    {label:'Ayesha', value: 'ayesha'},
+    {label:'Ryan', value: 'ryan'},
+  ]}
+  label='People who are awesome'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList if there are fewer than 4 items in the list and there is space to display all options. Use RadioButtons instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'No', value: 'no'},
+    {label:'Absolutely not', value: 'noagain'},
+  ]}
+  label='Is this a good use of a SelectList?'
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the same type of selection for a group of items. An example of this might be a filter bar. If some items could use SelectList and some items need to use Dropdown, use Dropdown for all the items in the group."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='Filter 1'
+  />
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv21'},
+      {label:'Filter value 2', value: 'fv22'},
+    ]}
+    placeholder='Filter 2'
+  />
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Mix Dropdown and SelectList in a group of items."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='SelectList'
+  />
+  <Button
```

Comment:
This should probably include an actual Dropdown

Reply:
Done!

Reply:
I'm still not seeing a Dropdown here?

Reply:
Really? It's working on my latest build.

Reply:
Now I see it! Nice!